Source/Pawnmorphs/Esoteria/PawnTransferUtilities.cs

```diff
@@ -308,7 +308,15 @@ public static void TransferFavor([NotNull] Pawn pawn1, [NotNull] Pawn pawn2)
                 health2.AddHediff(newHediff);
```

Comment:
Hediff_Psylink has a public field called "suppressPostAddLetter"
though a bit fiddly, you could try set that to false between making and adding the hediff if psylink
that might well get rid of the last letter